### PR TITLE
Add cogs QR Code Generator

### PR DESCRIPTION
**Pull request by GaniyaMustafa**

### CODE REVIEW

File: cogs/qrcode.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        generate(str(ctx.author.id), sub1)
+        with open(os.path.join(os.getcwd(), "src") + "/" + str(ctx.author.id)+".png", 'rb') as f:
+            picture = discord.File(f)
+            await ctx.channel.send(file=picture)
+        
+    
+
+def generate(name, subject):
+    global qr, photo, filename, save_dir, timestampStr1,type1
+    version, level, qr = myqr.run( subject, version=1, level='H', colorized=True, contrast=1.0, brightness=1.0, save_name=name+".png", save_dir=os.path.join(os.getcwd(), "src"))
```
Comment (harizMunawar) [inline cogs/qrcode.py:22]: This will hurt the repl.it storage. It will be great if the image is not saved or deleted after message sent

**Review by harizMunawar — COMMENTED** [review]
Please ignore the src folder or make sure the image is deleted directly after QR message is sent

File: cogs/qrcode.py (hunk 1 above)
Comment (Shaddamah) [reply cogs/qrcode.py:22]: consider using Imgur API [https://api.imgur.com/endpoints/image/](https://api.imgur.com/endpoints/image/)

**Review by Shaddamah — APPROVED** [review]

**Review by Shaddamah — CHANGES_REQUESTED** [review]
Please don't save the image on the storage, consider using the Imgur API [https://api.imgur.com/endpoints/image/](https://api.imgur.com/endpoints/image/)

**Review by harizMunawar — APPROVED** [review]

**Review by Shaddamah — APPROVED** [review]

**Comment by Shaddamah** [comment]
Merged! Great job

**Pull request merged**